bin/node/runtime/src/weights/pallet_balances.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Weights for the Balances Pallet
```

Comment:
usually I like to have doc telling when it is generated through script, (but anyway this is not part of this PR)

Reply:
maybe it could even say which commit of pallet-balance it uses, or its version (I didn't thought deep but could be easier to have some information about when we must update it)

Reply:
yes, good ideas. can do in a follow up